reference/managed-delivery/resource-status/index.md

```diff
@@ -0,0 +1,160 @@
+
+If Spinnaker detects a drift from the declarative configuration in the future, the resource will change status to [Diff](/reference/managed-delivery/resource-status/#diff). Once Spinnaker starts taking automatic action to correct the drift, the status will change to [Actuating](/reference/managed-delivery/resource-status/#actuating).
+
+While Spinnaker is managing a resource, manual changes made via the UI or the API will be automatically reversed ("stomped") in favor of the declarative configuration.
+
+### Diff
+{%
+  include
+  figure
+  image_path="./resource-status-diff.png"
+  caption="Example of a resource with the Diff status"
+%}
+
+When a resource has a Diff status, it means Spinnaker detected a drift from the declarative configuration but hasn't taken any action yet.
+
+Once Spinnaker starts taking automatic action to correct the drift, the status will change to [Actuating](/reference/managed-delivery/resource-status/#actuating).
+
+While Spinnaker is managing a resource, manual changes made via the UI or the API will be automatically reversed ("stomped") in favor of the declarative configuration.
+
+### Actuating
+{%
+  include
+  figure
+  image_path="./resource-status-actuating.png"
+  caption="Example of a resource with the Actuating status"
+%}
+
+When a resource has an Actuating status, it means Spinnaker detected a drift from the declarative configuration and automatic action is in progress to resolve it. You can go to the Tasks view in your app to see the actions Spinnaker is taking on your behalf, and whether they succeeded or failed.
+
+If automatic actions successfully resolve the drift, the status will change to [Happy](/reference/managed-delivery/resource-status/#happy). If automatic actions don't help to resolve the drift, the status will change to [Unhappy](/reference/managed-delivery/resource-status/#unhappy).
+
+While Spinnaker is managing a resource, manual changes made via the UI or the API will be automatically reversed ("stomped") in favor of the declarative configuration.
+
+### Paused
+{%
+  include
+  figure
+  image_path="./resource-status-paused.png"
+  caption="Example of a resource with the Paused status"
+%}
+
+When a resource has a Paused status, it means Spinnaker is configured to continuously manage the resource but you've chosen to temporarily pause management. While management is paused Spinnaker won't check for drift from the declarative configuration, and it won't take any automatic actions. You can go to the Config view in your app to pause and resume management across the entire application.
+
+When you resume management, Spinnaker will check for drift from the declarative configuration. If a drift is detected, the status will change to [Diff](/reference/managed-delivery/resource-status/#diff). If no drift is detected, the status will change to [Happy](/reference/managed-delivery/resource-status/#unhappy).
+
```

Comment:
Is it worth adding a line here about how, in the future, we want to show you want resources we will act on? and, in the even more future, what actions we will take? I don't want someone to read this and think "wow that's a shitty experience" - I just want to set expectations where things will grow and change and get better.

It might be too distracting to put that here though.

Reply:
Never mind, i see you plan to write more words later. 👍